packages/TelegramClient-Tests.package/TCTUITest.class/instance/assertNotReading.in..st

```diff
@@ -0,0 +1,13 @@
+asserting
+assertNotReading: aString in: aSuperMorph
+	
+	"TODO: How to negate assertions?"
+	aSuperMorph findDeepSubmorphThat: 
+		[:submorph |
+			(submorph isKindOf: TextMorph)
+				and: [submorph visible]
+					and: [submorph text asString includesSubstring: aString]]
+		ifAbsent: 
+			[^nil].
+	
+	AssertionFailure signal: 'Text ' , aString , ' found in ' , aSuperMorph class asString
```

Comment:
How about this? (You can just drag it into the image)

[TCTUITest-asserting.st.zip](https://github.com/hpi-swa-teaching/TelegramClient/files/6566018/TCTUITest-asserting.st.zip)

In order to then use assertNotReading or assertReading, you just have to supply an MTFMorphWrapper, such as `self subject` or `self subject createWrapperFor: {aSuperMorph}`. Would be much cleaner I think 🙂

Reply:
Thank you :) We have implemented your changes